api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -153,6 +154,24 @@ message ExtAuthz {
   // :ref:`destination<envoy_v3_api_field_service.auth.v3.AttributeContext.destination>`.
   // The labels will be read from :ref:`metadata<envoy_v3_api_msg_config.core.v3.Node>` with the specified key.
   string bootstrap_metadata_labels_key = 15;
+
+  // Check request to authorization server will include the client request headers that have a correspondent match
```

Comment:
I don't think we should change the existing behavior. I think if this new message isn't set, then the previous behavior of sending all headers should apply.